benchmarks/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
+FetchContent_MakeAvailable(gtest)
+
+find_package(gtest)
```

Comment:
Doesnt `FetchContent_MakeAvailable` already does `find_package`?

Reply:
Oh right, I will remove it again